data_tools/README.md

```diff
@@ -3,4 +3,4 @@ Firebase console > project settings > service accounts
 Add the Firebase Service Account secret as `serviceAccount.json` into the current folder
 
 # Write sample
-`python3 writeSample.py </path/to/fb_secret.json> </path/to/fb_const.json> <themes|filters>`
+`python3 writeSample.py </path/to/fb_secret.json> </path/to/fb_const.json> <themes|filters|people>`
```

Comment:
I think it's a fb_service_account.json isn't it, rather than a secret?

Reply:
Fixed